build/reference/9074DuplicateParallelStateMachineNameInParallelRegion.txt

```diff
@@ -0,0 +1,24 @@
+E074 Duplicate Parallel State Machine Name in Parallel Region
```

Comment:
@vahdat-ab please see my comments for examples:  
- 487_parallelStateMachines_sameRegionSameNames.ump
- 487_parallelStateMachines_sameRegionSameNames_2.ump
- E074DuplicateParallelStateMachineNameInParallelRegion1.ump
- E074DuplicateParallelStateMachineNameInParallelRegion2.ump